### PR TITLE
Add show Instructor, edit/delete Instructor

### PR DESCRIPTION
**Pull request by Konfeta3-33**

### CODE REVIEW

File: app/controllers/instructors_controller.rb [hunk 1]
```diff
@@ -19,10 +27,34 @@ def create
...
+    if @instructor.update instructor_params
+      flash[:notice] = "Instructor #{@instructor.name} #{@instructor.surname} updated!"
+      redirect_to instructor_path
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @instructor.destroy
```
Comment (skyksandr) [inline app/controllers/instructors_controller.rb:40]:
```ruby
if @instructor.destroy
  ...
else
  # show errors
end
```

you also need to delete `user` associated with instructor

Comment (Konfeta3-33) [reply app/controllers/instructors_controller.rb]: Done

File: app/controllers/instructors_controller.rb [hunk 2]
```diff
@@ -37,9 +37,12 @@ def update
...
 
   def destroy
-    @instructor.destroy
-    redirect_to instructors_path, status: :see_other
-    flash[:alert] = "Instructor #{@instructor.name} #{@instructor.surname} deleted!"
+    if @instructor.destroy
+      redirect_to instructors_path, status: :see_other
+      flash[:alert] = "Instructor #{@instructor.name} #{@instructor.surname} deleted!"
+    else
+      render file: 'public/404.html', status: :not_found
```
Comment (skyksandr) [inline app/controllers/instructors_controller.rb:44]: This doesn't seem correct. better render `render :edit` with `status: :unprocessable_entity` and `flash`

Comment (Konfeta3-33) [reply app/controllers/instructors_controller.rb:44]: ok. Done

**Pull request merged**